docs/content/guides/cell-features/merge-cells/merge-cells.md

```diff
@@ -66,6 +66,58 @@ To initialize Handsontable with predefined merged cells, provide merged cells de
 
 :::
 
+## Optimizing rendering of the wide/tall merged cells
```

Comment:
Maybe: Extremely wide and tall merged cells

Reply:
I would leave it as is. Those optimizations work not only for extremally wide/tall merged cells (e.g., row span > 1000) but also make sense for "regular" sizes (e.g., rowspan=50).